WA.Pizza.Web/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "Default": "Server=localhost,1433;Database=WA.PizzaDB;User=sa;Password=#sql-pass22_"
```

Comment:
Server=mssql
as your service name in compose

Reply:
Replaced